cmr-exchange/authz/src/cmr/authz/token.clj

```diff
@@ -63,14 +63,15 @@
   (let [url (str base-url token-info-resource)
         data (str "id=" token)]
     (log/trace "Making token-info query to ECHO REST:" url)
+    (log/debug "Value from authz token:" data)
```

Comment:
Has the token been partially redacted?  You are not showing the entire token are you?

Reply:
That was just for debug on my end, I'll be removing that with other changes related to some of the code changes I had to make